spec/permutations_properties.cry

```diff
@@ -0,0 +1,42 @@
+import permutations
+
+// ==========
+// Properties
+// ==========
+
+//  We'll just prove up to 2^^4096 --- that seems plenty and is still fast
+// Function `is_power_of_two` should be true for all powers of two
+property check_is_power_of_two = all is_power_of_two`{4096} [ 2^^n | n <- [0 .. 4096-1]]
```

Comment:
We could utilize the new `:check-dockstring` feature in `cryptol` by wrapping the `:prove, :check` etc commands in docstrings.

Reply:
Oh, nice! Where do I find the documentation about how to run these? Or is there a help for each command? I'll try this out later today.